airstack-modules/modules/airstack/common/index.ts

```diff
@@ -72,18 +74,22 @@ AIR_CHAIN_ID_MAP.set("xdai", "100");
 
 export function getNetwork(network: string): string {
   const value = AIR_NETWORK_MAP.get(network);
-  const result: string = value !== null ? value : "unknown";
-  return result;
+  if (value != null) {
+    return value!;
+  }
+  throw new Error("Network not supported");
 }
 
 export function getChainId(): string {
   let network = dataSource.network();
   const value = AIR_CHAIN_ID_MAP.get(network);
-  const result: string = value !== null ? value : "unknown";
-  return result;
+  if (value != null) {
+    return value!;
```

Comment:
Thesis because compiler throws error even after null check is done.
![Screenshot 2023-02-16 at 11 13 58 PM](https://user-images.githubusercontent.com/42062368/219445808-0f3046f4-1539-4c96-a1c7-43f52e1421f7.png)